src/libs/actions/Policy/Tag.ts

```diff
@@ -531,7 +531,7 @@ function renamePolicyTaglist(policyID: string, policyTagListName: {oldName: stri
     API.write(WRITE_COMMANDS.RENAME_POLICY_TAG_LIST, parameters, onyxData);
 }
 
-function setPolicyRequiresTag(policyID: string, requiresTag: boolean) {
+function setPolicyRequiresTag(policyID: string, requiresTag: boolean, tagListIndex: number) {
```

Comment:
The optimistic data (and other onyx data) is updating the per workspace tag requirement and not per tag list

Reply:
Also the BE is updating everything (per workspace tag requirement + every per tag list requirement)


https://github.com/Expensify/App/assets/16493223/054440e6-0fd6-498f-98a3-63a9bd211acf